core/schema/envfile.go

```diff
@@ -0,0 +1,90 @@
+package schema
+
+import (
+	"context"
+
+	"github.com/dagger/dagger/core"
+	"github.com/dagger/dagger/dagql"
+)
+
+type envfileSchema struct{}
+
+var _ SchemaResolvers = &envfileSchema{}
+
+func (s envfileSchema) Install(srv *dagql.Server) {
+	dagql.Fields[*core.Query]{
+		dagql.Func("envFile", s.newEnvFile).
```

Comment:
Don't want to have a super extended bikeshed, but what do you think about instead calling this `EnvVariables` / `EnvVariableSet` / etc.

IMO, I like the idea of having a function that can take a collection of various env vars, potentially being to bulk-inject them into a function (using a future `Container.withEnvVariables`.

That way, it's detached from the concept of an underlying file, while still being able to be parsed using `File.asEnvFile`, or converted to one using `EnvVariables.file` (or maybe even `EnvVariables.asFile`).

(Also the stutter of `EnvFile.file` feels kind bleh, another motivator for renaming the type/constructor)